korean/coco/coco.md

```diff
@@ -0,0 +1,3 @@
+핫초코를 사러 나온 당신... 돈은 있나요? 핫초코를 어디서 살건가요? 
+돈이 없으면 뭘 어쩌겠어? [다시 방으로 돌아간다](../marshmallow.md) 집에서 잠이나 자!
+돈이 없으면 핫초코를...직접 찾아나서는거! [핫초코를 찾아 여행](../explore-outside/find-coco.md) 
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.